### DISCUSSION TITLE
What's the right way to handle "eBay API request error"?

### DISCUSSION THREAD

**Discussion by nrathi** [hendt/ebay-api#147] [Q&A]
It's the fallback here: https://github.com/hendt/ebay-api/blob/d6d165e176b34bc444b4e7c37fdc0d9ca01927c9/src/errors/index.ts#L69

Is the appropriate response to retry a few times with exponential delays, or fail gracefully?

**Comment by Clarry100** [comment]
I've been using exponential delays to retry then if it still fails report a retry limit exceed back to the user. But I don't know if/not this would be the right way to do it. But it works for me.
